app/models/conversion_host/configurations.rb

```diff
@@ -41,12 +41,20 @@ def enable(params)
       params = params.symbolize_keys
       _log.info("Enabling a conversion_host with parameters: #{params}")
 
-      params.delete(:task_id)     # In case this is being called through *_queue which will stick in a :task_id
-      params.delete(:miq_task_id) # The miq_queue.activate_miq_task will stick in a :miq_task_id
+      task_id = params.delete(:task_id)         # In case this is being called through *_queue which will stick in a :task_id
+      miq_task_id = params.delete(:miq_task_id) # The miq_queue.activate_miq_task will stick in a :miq_task_id
+      task_id ||= miq_task_id
 
       vddk_url = params.delete(:param_v2v_vddk_package_url)
 
       conversion_host = new(params)
+
+      unless task_id.nil?
```

Comment:
Probably change this to `if task_id.present?` in case it comes through as an empty param instead of nil. I've come to be paranoid about such things when dealing with http.